docs/development/LedStrip.md

```
@@ -144,35 +108,153 @@ The datasheet can be found here: http://www.adafruit.com/datasheets/WS2812.pdf
-If used with zero arguments it prints out the LED configuration which can be copied for future reference.
+#### Advanced LED configuration
 
-Each LED is configured using the following template: `x,y:ddd:mmm:cc`
+The configuration values for each LED can be displayed in the CLI using the `led` command.
 
-`x` and `y` are grid coordinates of a 0 based 16x16 grid, north west is 0,0, south east is 15,15
-`ddd` specifies the directions, since an LED can face in any direction it can have multiple directions. Directions are:
+The `led` command with no arguments prints out the current LED configuration, which can be copied for future reference.
+
+Otherwise, `led` expects two arguments - a zero-based LED index number, a space, and then a sequence of parameters in the form:
+` index x,y:ddd:mmm:cc`, where:
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)